README.md

```diff
@@ -1,73 +1,67 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Anime-World
 
-> Description the project.
+> "Anime-World" is a website show list of animes and show different facts about them.
 
+![Homepage](./application.png)
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+Design From [Behance](https://www.behance.net/gallery/31579789/Ballhead-App-%28Free-PSDs%29)
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Built With
 
+- HTML
+- CSS
+- JAVASCRIPT
+- REACT
+- REDUX
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+Start by cloning the repository.
 
+`git clone https://github.com/thermiee/React-Capstone.git`
 
-To get a local copy up and running follow these simple example steps.
+Navigate to the created folder.
 
-### Prerequisites
+`cd React-Capstone`
 
-### Setup
+Install all the dependencies.
 
-### Install
+`npm install`
 
-### Usage
+Run the live server
 
-### Run tests
+`npm run start`
 
-### Deployment
+Go to [Localhost](http://localhost:3000)
```

Comment:
- [ ] as part of the project you are also required to deploy this website. please ensure you deploy this project to either GH pages, Netlify or Heroku.

Reply:
- [ ] I'd suggest you either remove this or add your working deploy link here🤷‍♂️